pipeline.py

```diff
@@ -21,7 +21,7 @@
 
 import nn
 
-HOME_DIR = '/scratch/users/johmathe'
+HOME_DIR = '/scratch/users/nmiolane'
```

Comment:
can we have something like a config.py where we define this and not have to send it for review everytime?